<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdditionalPaymentView</class>
 <widget class="QWidget" name="AdditionalPaymentView">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>759</width>
    <height>727</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	background-color: #191A19;
	color: white;
	font-family: Sans-serif;
	font-size: 14px;
}
QPushButton {
	border: 1px solid #606060;
	border-radius: 25px;
}

QPushButton::hover {
	background-color: #444444;
}

QPushButton::pressed {
	background-color: #282828;
}

QSpinBox {
	border: 1px solid #606060;
	border-radius: 25px;
	padding-left: 10px;
}

QDoubleSpinBox {
	border: 1px solid #606060;
	border-radius: 25px;
	padding-left: 10px;
}

QComboBox {
	border: 1px solid #606060;
	border-radius: 25px;
	padding-left: 10px;
}
QComboBox::item::selected {
	background-color: #444444;
}

QDateEdit {
	border: 1px solid #606060;
	border-radius: 20px;
	padding-left: 10px;
}</string>
  </property>
  <widget class="QPushButton" name="homeBtn">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>640</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: #374045;
}

QPushButton::hover {
	background-color: #3A4750;
}

QPushButton::pressed {
	background-color: #303841;
}</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="applyBtn">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>640</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: #E16428;
}

QPushButton::hover {
	background-color: #ff8000;
}

QPushButton::pressed {
	background-color: #b35900;
}</string>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addBtn">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>510</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: #374045;
}

QPushButton::hover {
	background-color: #3A4750;
}

QPushButton::pressed {
	background-color: #303841;
}</string>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="replInputWidget">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>70</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans-serif</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>1000000000</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>1</number>
   </property>
  </widget>
  <widget class="QPushButton" name="delBtn">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>510</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: #374045;
}

QPushButton::hover {
	background-color: #3A4750;
}

QPushButton::pressed {
	background-color: #303841;
}</string>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QComboBox" name="replTypeWidget">
   <property name="geometry">
    <rect>
     <x>24</x>
     <y>70</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans-serif</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:0;</string>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <property name="currentText">
    <string>One-time</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="maxVisibleItems">
    <number>7</number>
   </property>
   <property name="insertPolicy">
    <enum>QComboBox::InsertAtBottom</enum>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
   </property>
   <property name="minimumContentsLength">
    <number>0</number>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
   <property name="duplicatesEnabled">
    <bool>false</bool>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="modelColumn">
    <number>0</number>
   </property>
   <item>
    <property name="text">
     <string>One-time</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Monthly</string>
    </property>
   </item>
  </widget>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>264</x>
     <y>70</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 0;</string>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="correctionMode">
    <enum>QAbstractSpinBox::CorrectToPreviousValue</enum>
   </property>
   <property name="showGroupSeparator" stdset="0">
    <bool>false</bool>
   </property>
   <property name="displayFormat">
    <string>dd.MM.yyyy</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
   <property name="timeSpec">
    <enum>Qt::LocalTime</enum>
   </property>
   <property name="date">
    <date>
     <year>2022</year>
     <month>6</month>
     <day>15</day>
    </date>
   </property>
  </widget>
  <widget class="QListView" name="listView">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>180</y>
     <width>491</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
